atomicwrites/__init__.py

```diff
@@ -131,18 +139,19 @@ def __init__(self, path, mode='w', overwrite=False):
         if 'w' not in mode:
             raise ValueError('AtomicWriters can only be written to.')
 
-        self._path = path
-        self._mode = mode
-        self._overwrite = overwrite
+        self._path = _path_to_unicode(path)  # type: Text
```

Comment:
Also in principle I don't want to force-convert something here just for the type annotations.